packages/components/src/draggable/index.js

```diff
@@ -148,6 +149,18 @@ class Draggable extends Component {
 
 	render() {
 		const { children, className } = this.props;
+		if ( typeof children === 'function' ) {
+			return children( {
+				onDraggableStart: this.onDragStart,
+				onDraggableEnd: this.onDragEnd,
+			} );
+		}
+
+		deprecated( 'wp.components.Draggable as a DOM node drag handle', {
+			version: 3.8,
```

Comment:
This should be `3.10` because we need to only wait two releases with the deprecation message and I think we should wait until the `3.7` release to merge this PR.

Reply:
We should also add `plugin: 'Gutenberg',`

Reply:
d6e0152